Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml

```diff
@@ -561,3 +561,17 @@
   - type: Item
     heldPrefix: gold
     
+
+- type: entity
+  parent: IDCardStandard
```

Comment:
the idea to inherit from clown is future proofing so that whatever gets added to clowns in the future applies to cluwnes

Reply:
> the idea to inherit from clown is future proofing so that whatever gets added to clowns in the future applies to cluwnes

I kinda get that but there was things in the IDCardClown that i didnt want in the Cluwne id such as their job role and access. I am not sure how to ignore certain elements from a parent. How do i do this so the cluwne id doesnt inherit the clowns access on its id card?

Reply:
disregard then